src/vm/core.zig

```diff
@@ -482,6 +482,46 @@ pub fn mulOperands(
     return relocatable.fromFelt(op_0_felt.mul(op_1_felt));
 }
 
+/// Attempts to deduce `op1` and `res` for an instruction, given `dst` and `op0`.
+///
+/// # Arguments
+/// - `inst`: The instruction to deduce `op1` and `res` for.
+/// - `dst`: The destination of the instruction.
+/// - `op0`: The first operand of the instruction.
+///
+/// # Returns
+/// - `Tuple`: A tuple containing the deduced `op1` and `res`.
+pub fn deduce_op1(
```

Comment:
camelCase for fns
```suggestion
pub fn deduceOp1(
```

Reply:
Rust haunting me

Reply:
we can keep our fingers crossed for that proposal for snake_case functions to be accepted and implemented, because I do agree that snake_case fns are probably better :)